### PR TITLE
Update draft-tigress-requirements.md

### PR DESCRIPTION
**Pull request by dimmyvi**
Update based on Manuel's review

### CODE REVIEW

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -78,8 +78,9 @@ Tigress:
 
 --- abstract
 
-This document describes the use cases necessitating the secure transfer of digital credentials. The document also comprises a proposal, and defines requirements and scope.
-
+This document describes the use cases necessitating the secure transfer of digital credentials between two devices. 
+Secure credentials may represent a digital key to a hotel room, a digital key to a door lock in a house or a digital key to a car. 
+The document also comprises a proposal, and defines requirements and scope.
```
Comment (ManuelGersterB) [inline draft-tigress-requirements.md:83]: @dimmyvi: I really like the new sentence in line 82, but unfortunately the sentence in line 83 seems to be lost a bit now.
Maybe we can add line 83 to line 81, e. g.:
"This document describes the use cases necessitating the secure transfer of digital credentials between two devices and defines general assumptions, requirements and the scope of the corresponding Internet-draft."

**Comment by ManuelGersterB** [comment]
Just now I have observed one little point. The ":" should be removed here: 
image

File: draft-tigress-requirements.md (hunk 1 above)
Comment (dimmyvi) [reply draft-tigress-requirements.md]: Done.

File: draft-tigress-requirements.md [hunk 2]
```diff
@@ -57,6 +57,12 @@ author:
     name: Nick Sha
     organization: Alphabet Inc
     email: nicksha@google.com
+ -
+    ins: M. Gerster
+    name: Manuel Gerster
+    organization: Mercedes Benz
```
Comment (ManuelGersterB) [inline draft-tigress-requirements.md:63]: @dimmyvi: Thx for adding me :) Just one request: Could you please change the organization to "Mercedes-Benz AG"?

**Pull request merged**